### PR TITLE
get language from request

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
@miltontony @codiebeulaine ready for review

**Comment by codiebeulaine** [comment]
@saeedmb8 should we maybe test this?

File: google_analytics/utils.py [hunk 1]
```diff
@@ -114,10 +118,11 @@ def build_ga_params(request, account, path=None, event=None, referer=None, title
     # construct the gif hit url
     ga_url = "http://www.google-analytics.com/collect"
     utm_url = ga_url + "?&" + urllib.urlencode(params)
-
+    locale = get_locale_code(get_language_from_request(request))
+    
     return {'utm_url': utm_url,
             'user_agent': user_agent,
-            'language': meta.get('HTTP_ACCEPT_LANGUAGE', ''),
+            'language': locale,
```
Comment (miltontony) [inline google_analytics/utils.py]:
```python
'language': locale or settings.LANGUAGE_CODE
```

File: google_analytics/utils.py [hunk 2]
```diff
@@ -114,10 +118,11 @@ def build_ga_params(request, account, path=None, event=None, referer=None, title
     # construct the gif hit url
     ga_url = "http://www.google-analytics.com/collect"
     utm_url = ga_url + "?&" + urllib.urlencode(params)
-
+    locale = get_locale_code(get_language_from_request(request))
```
Comment (miltontony) [inline google_analytics/utils.py]: we don't need `get_locale_code` here

**Comment by miltontony** [comment]
👍

**Pull request merged**